apm-sniffer/config/logconfig.yaml

```diff
@@ -0,0 +1,12 @@
+- name: "log4j"
+  packages:
+    - "*"
+  level: "error"
+- name: "log4j2"
+  packages:
+    - "*"
+  level: "error"
+- name: "logback"
+  packages:
+    - "*"
+  level: "error"
```

Comment:
do we really want to introduce YAMLs into the agent plugin? It seems this kind of file content can be represented in a simple format so that we don't need to shade a new dependency 


```properties
log4j.packages=foo.bar,foo.baz  # csv
log4j.level=error
log4j2.packages=*
log4j2.level=error
```

Reply:
2 things, 
1. Why use YAML
1. If use properties, the agent core has provided the full capability to extend your own format. Recommend to adopt that.

Reply:
ok,I will adopt it.